### PR TITLE
add book and update book

### PR DESCRIPTION
**Pull request by SalSoloman**

### CODE REVIEW

File: routes/api.js [hunk 1]
```diff
@@ -12,9 +11,27 @@ app.get('/', function (req, res) {
...
-  .then( results => {
-    res.send(deleted)
+    .then( results => {
+      res.send(deleted)
   })
 })
 
+app.post('/', function (req, res){
+  const body = req.body
+  console.log(body)
```
Comment (shakalee14) [inline routes/api.js:21]: can we remove this console.log?

File: routes/api.js [hunk 2]
```diff
@@ -12,9 +11,27 @@ app.get('/', function (req, res) {
...
+
+app.post( '/add', function (req, res) {
+  const {title, description} = req.body
+  Books.add(title, description)
+    .then( results => {
+      res.status(200).json({data: results})
+    })
+})
+
+app.put()
```
Comment (shakalee14) [inline routes/api.js:33]: what is this put method doing here?

File: database/db.js [hunk 3]
```diff
@@ -10,11 +10,14 @@ const db = pgp(conectionString)
 
 const GET_ALL = "SELECT * FROM book"
 const DELETE_BOOK = "DELETE FROM book WHERE id = $1"
-
+const ADD_BOOK = "INSERT INTO book (title, description) VALUES ('$1', '$2') returning *;"
+const UPDATE_BOOK = "UPDATE book SET title='$1' WHERE id=$2;"
 
 const Books = {
```
Comment (shakalee14) [inline database/db.js:16]: clean.

File: routes/api.js (hunk 2 above)
Comment (SalSoloman) [reply routes/api.js:33]: I was in the process of writing it :)

File: database/db.js (hunk 3 above)
Comment (SalSoloman) [reply database/db.js:16]: sorry I didn't understand the commment, do I need to clean it or you're saying it clean as in it's good?

File: routes/api.js (hunk 1 above)
Comment (SalSoloman) [reply routes/api.js:21]: ok will do

**Pull request merged**